src/PostgREST/Version.hs

```diff
@@ -26,7 +26,7 @@ prettyVersion =
 docsVersion :: Text
 docsVersion
   | isPreRelease = "latest"
-  | otherwise    =  "v" <> (T.intercalate "." . map show . take 1 $ version)
+  | otherwise    =  "v" <> T.intercalate "." (take 1 version)
```

Comment:
Look enough going from the discussion on https://github.com/PostgREST/postgrest/pull/4168/files#r2171481594